README.md

```diff
@@ -39,6 +39,8 @@ Goal is to support the equivalent of:
 Multipart uploads are an important part of S3, however, they aren't simple to implement and have substantial performance
 and storage implications for a naive approach. As of now, this API does not attempt to provide that functionality.
 
+See [To disable multipart in clients](#disabling-multipart)
```

Comment:
How about ...

`See [Disabling Multipart](#disabling-multipart) for details.`

Reply:
Done.